app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -435,7 +435,7 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
             }
 
             @Suppress("ConstantConditionIf")
-            if (FeatureFlags.asFeatureWebChannelsDisabled) {
+            if (!FeatureFlags.asFeatureWebChannelsDisabled) {
```

Comment:
No action needed: confirmed that this `!` should have been there: https://github.com/mozilla-mobile/fenix/pull/7510/files#diff-bf0b57d467da0e3d5ef96c23fc828376R438